apps/easypid/src/app/authenticate.tsx

```diff
@@ -15,17 +23,19 @@ import { useResetWalletDevMenu } from '../utils/resetWallet'
 export default function Authenticate() {
   useResetWalletDevMenu()
 
+  const toast = useToastController()
   const secureUnlock = useSecureUnlock()
   const pinInputRef = useRef<PinDotsInputRef>(null)
   const [isInitializingAgent, setIsInitializingAgent] = useState(false)
   const isLoading =
     secureUnlock.state === 'acquired-wallet-key' || (secureUnlock.state === 'locked' && secureUnlock.isUnlocking)
 
+  // biome-ignore lint/correctness/useExhaustiveDependencies: canTryUnlockingUsingBiometrics not needed
   useEffect(() => {
     if (secureUnlock.state === 'locked' && secureUnlock.canTryUnlockingUsingBiometrics) {
       secureUnlock.tryUnlockingUsingBiometrics()
     }
-  }, [secureUnlock])
+  }, [])
```

Comment:
This doesn't seem correct. What are you trying to do?

Reply:
Currently, it prompts me for me biometrics right away, and if you cancel it prompts again until you reach 3.

Now it prompts once, if it fails or gets cancelled it stops, and you can prompt it again by pressing the biometrics button on the pinpad.

Reply:
Hmm okay. I see. We should still pass `secureUnlock.state` right?